src/key-derivation/README.md

```diff
@@ -0,0 +1,9 @@
+# Key Derivation Functions
+
+[Key derivation functions] provide a way to take an input such as a password, and crate an output of
+specified length. This output can then be used as e.g. a key for an encryption algorithm.
```

Comment:
Can you revise the sentence to not use e.g.?  Perhaps: "used, for example, as the key in"

Reply:
I reworded, please take another look

Reply:
looks good, thanks!